src/containers/my-resources/lessons-container/LessonsContainer.constants.tsx

```diff
@@ -0,0 +1,78 @@
+import ListAltIcon from '@mui/icons-material/ListAlt'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/containers/my-resources/lessons-container/LessonsContainer.styles'
+import { authRoutes } from '~/router/constants/authRoutes'
+import {
+  AdditionalPropsInterface,
+  Lesson,
+  Offer,
+  RemoveColumnRules,
+  SortEnum,
+  TableColumn
+} from '~/types'
+import { createUrlPath, getFormattedDate } from '~/utils/helper-functions'
+
+export const columns: TableColumn<Lesson>[] = [
+  {
+    label: 'myResourcesPage.lessons.title',
+    field: 'title',
+    calculatedCellValue: (
+      item: Lesson,
+      { navigate }: AdditionalPropsInterface
+    ) => {
+      const navigateLesson = () => {
+        navigate(createUrlPath(authRoutes.myResources.myLesson.path, item._id))
+      }
+
```

Comment:
i think, you can move this function above and it should accept itemId and navigate function and then pass it to onClick={() => navigateLesson(item._id, navigate)}